client/my-sites/people/people-profile/index.jsx

```diff
@@ -222,52 +223,52 @@ class PeopleProfile extends Component {
 		);
 	};
 
-	renderSubscribedDate = () => {
-		if ( ! this.props.user || ! this.props.user.date_subscribed ) {
+	const renderSubscribedDate = () => {
+		if ( ! user || ! user.date_subscribed ) {
 			return;
 		}
 
 		return (
 			<div className="people-profile__subscribed">
-				{ this.props.translate( 'Since %(formattedDate)s', {
+				{ translate( 'Since %(formattedDate)s', {
 					context: 'How long a user has been subscribed to a blog. Example: "Since Sep 16, 2015"',
 					args: {
-						formattedDate: this.props.moment( this.props.user.date_subscribed ).format( 'll' ),
+						formattedDate: moment( user.date_subscribed ).format( 'll' ),
 					},
 				} ) }
 			</div>
 		);
 	};
 
-	isFollowerType = () => {
-		return this.props.user && ! this.props.user.roles && this.props.user.date_subscribed;
+	const isFollowerType = () => {
+		return user && ! user.roles && user.date_subscribed;
 	};
 
-	render() {
-		const { user } = this.props;
+	const classes = classNames( 'people-profile', {
+		'is-placeholder': ! user,
+	} );
 
-		const classes = classNames( 'people-profile', {
-			'is-placeholder': ! user,
-		} );
-
-		return (
-			<div className={ classes }>
-				<div className="people-profile__gravatar">
-					<Gravatar user={ user } size={ 72 } />
-				</div>
-				<div className="people-profile__detail">
-					{ this.renderNameOrEmail() }
-					{ this.renderLogin() }
-					{ this.isFollowerType() ? this.renderSubscribedDate() : this.renderRole() }
-				</div>
+	return (
+		<div className={ classes }>
+			<div className="people-profile__gravatar">
+				<Gravatar user={ user } size={ 72 } />
 			</div>
-		);
-	}
-}
-
-export default connect( ( _state, { siteId, user } ) => {
-	const externalContributors = ( siteId && requestExternalContributors( siteId ).data ) || [];
-	return {
-		isExternalContributor: externalContributors.includes( user?.linked_user_ID ?? user?.ID ),
-	};
-} )( localize( withLocalizedMoment( PeopleProfile ) ) );
+			<div className="people-profile__detail">
+				{ renderNameOrEmail() }
+				{ renderLogin() }
+				{ isFollowerType() ? renderSubscribedDate() : renderRole() }
+			</div>
+		</div>
+	);
+};
+
+PeopleProfile.propType = {
+	siteId: PropTypes.number.isRequired,
+	translate: PropTypes.func.isRequired,
+	moment: PropTypes.func.isRequired,
```

Comment:
The `translate` and `moment` props are no longer there, they've been replaced by hook calls inside the component function.